src/Transport/Receiving/MessagePump.cs

```diff
@@ -196,7 +196,7 @@ async Task OnProcessMessage(ProcessMessageEventArgs arg)
             }
             catch (Exception ex) when (ex.IsCausedBy(messageProcessingCancellationTokenSource.Token))
             {
-                Logger.Debug("Message processing canceled.", ex);
+                Logger.Debug($"Processing of the message with id '{messageId}' canceled.", ex);
```

Comment:
Cancelled

Reply:
We standardized on American English

https://www.merriam-webster.com/grammar/canceled-or-cancelled